app/controllers/shopify_app/callback_controller.rb

```diff
@@ -23,7 +23,16 @@ def callback
 
       return respond_with_user_token_flow if start_user_token_flow?(api_session)
 
-      perform_post_authenticate_jobs(api_session)
+      if ShopifyApp::VERSION < "23.0"
```

Comment:
Nice that we can just compare it this way, makes this very easy to understand :)